_candidates/jeff-sheehy.md

```diff
@@ -1,39 +1,71 @@
 ---
-id: 28
+id: 47
 name: Jeff Sheehy
 photo_url: null
-website_url: "http://jeffsheehy.org/"
+website_url: null
```

Comment:
He has a row in the candidates spreadsheet (on the "All Candidates") tab now, that can be used to populate info like this.